bin/pulsar-perf

```diff
@@ -73,12 +73,11 @@ pulsar_help() {
     cat <<EOF
 Usage: pulsar <command>
 where command is one of:
-    produce                        Run a producer
-    consume                        Run a consumer
-    monitor-simple-load-manager    Continuously receive broker data when using SimpleLoadManagerImpl
-    monitor-modular-load-manager   Continuously receive broker data when using ModularLoadManagerImpl
-    simulation-client              Run a simulation server acting as a Pulsar client
-    simulation-controller          Run a simulation controller to give commands to servers
+    produce                 Run a producer
+    consume                 Run a consumer
+    monitor                 Continuously receive broker data and/or load reports
```

Comment:
A think `monitor-brokers` would be good, since it is really just printing out broker data in general.